frontend/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@code-dot-org/frontend",
+  "private": true,
+  "scripts": {
+    "build": "turbo build",
+    "dev": "turbo dev",
+    "lint": "turbo lint",
+    "format": "prettier --write \"**/*.{ts,tsx,md}\""
+  },
+  "devDependencies": {
+    "@storybook/test-runner": "^0.19.1",
+    "prettier": "^3.2.5",
+    "rimraf": "^6.0.1",
+    "turbo": "^2.3.0",
+    "typescript": "5.5.4"
+  },
+  "engines": {
+    "node": ">=22"
```

Comment:
It should force node 22 (yarn will fail), but you're right that I should add a `.nvmrc` that specifies node 22.  

I'll create a new task to add that file.

Reply:
I successfully `yarn install`-ed and don't have Node 22 installed.

Reply:
Interesting, looks like yarn 2+ [removed the check](https://github.com/yarnpkg/berry/issues/1177) but yarn 1 (classic) still has it. I recommend we keep the check in case yarn decides to add it in the future and it helps other tools in the future.

```
yarn install v1.22.22
warning Missing version in workspace at "/Users/admin/git-workspaces/code-dot-org/frontend/apps/design-system-storybook", ignoring.
[1/5] 🔍  Validating package.json...
warning Missing version in workspace at "/Users/admin/git-workspaces/code-dot-org/frontend/apps/design-system-storybook", ignoring.
error @code-dot-org/frontend@: The engine "node" is incompatible with this module. Expected version ">=22". Got "18.20.4"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```